packages/commonwealth/client/scripts/views/menus/create_content_menu.tsx

```diff
@@ -135,6 +137,7 @@ const getCreateContentMenuItems = (navigate): PopoverMenuItem[] => {
             onClick: () =>
               navigate('/new/proposal/:type', {
                 type: ProposalType.SubstrateTreasuryTip,
+                deferChain: false,
```

Comment:
I think this is the same thing as I commented before => adding `deferChain` here is pointless so please remove it. I am pretty sure that `type` does not work neither here and I think this would make much more sense:
```
navigate(`/new/proposal/${ProposalType.SubstrateTreasuryProposal}`)
```